core/templates/components/site-search/site-search.twig

```diff
@@ -0,0 +1,27 @@
+{#
+/**
+ * @file
+ * Global Site Searach Component
+ *
+ * A default link component.
+ *
+ * Available variables:
+ * - action: Action of the form field
+ * - method: Method of the form field
+ * - input_attributes: Additional attributes for the input field
+ * - placeholder: Placeholder text for the input field
+ * - button_attributes: Additional button attributes
+ * - search_icon_src: The href property of the icon image
+ * - additional_fields: Any additional fields, e.g., hidden input fields
+ *
+ */
+#}
+<div class="su-site-search" role="search">
```

Comment:
missing attributes and modifier class variables.

Reply:
Added